virtcontainers/hyperstart_agent.go

```diff
@@ -24,7 +24,7 @@ import (
 	"github.com/kata-containers/runtime/virtcontainers/pkg/hyperstart"
 	ns "github.com/kata-containers/runtime/virtcontainers/pkg/nsenter"
 	"github.com/kata-containers/runtime/virtcontainers/utils"
-	specs "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/opencontainers/runtime-spec/specs-go"
```

Comment:
Won't it break build?

Reply:
It won't. Better to leave it as is

Reply:
Maybe it's treat of `gland` IDE `gofmt` action. 

Leave it is better~